akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/TestProbe.scala

```diff
@@ -220,6 +220,11 @@ abstract class TestProbe[M] {
    */
   @InternalApi protected def expectMessageClass_internal[C](max: FiniteDuration, c: Class[C]): C
 
+  /**
+   * Receive one message of type `M` within `remainingOrDefault` as the `TestTimeFactor`.
+   */
+  def receiveOne(): M
+
```

Comment:
also add one with `max: Duration` parameter

Reply:
I originally had that but it conflicted with an existing. Will see what the problem was and add.